konfera/event/views.py

```diff
@@ -74,15 +88,50 @@ def dispatch(self, *args, **kwargs):
 
     def post(self, *args, **kwargs):
         context = self.get_context_data(**kwargs)
+        notify = getattr(django_settings, 'PROPOSAL_NOTIFY', getattr(settings, 'PROPOSAL_NOTIFY'))
+        template = EmailTemplate.objects.get(name='confirm_proposal')
 
         if context['speaker_form'].is_valid() and context['talk_form'].is_valid():
-            speaker_instance = context['speaker_form'].save()
+            speaker = context['speaker_form'].save()
             talk_instance = context['talk_form'].save(commit=False)
-            talk_instance.primary_speaker = speaker_instance
+            talk_instance.primary_speaker = speaker
             talk_instance.event = context['event']
             talk_instance.status = talk_instance.status or Talk.CFP
             talk_instance.save()
-            messages.success(self.request, self.message_text)
+
+            if notify:
```

Comment:
Yes you can. See https://github.com/pyconsk/django-konfera/blob/master/payments/tests/test_views.py#L27 .